pkg/remote/aws/route53_health_check_enumerator.go

```diff
@@ -0,0 +1,46 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+type Route53HealthCheckEnumerator struct {
+	repository repository.Route53Repository
+	factory    resource.ResourceFactory
+}
+
+func NewRoute53HealthCheckEnumerator(repo repository.Route53Repository, factory resource.ResourceFactory) *Route53HealthCheckEnumerator {
+	return &Route53HealthCheckEnumerator{
+		repo,
+		factory,
+	}
+}
+
+func (e *Route53HealthCheckEnumerator) SupportedType() resource.ResourceType {
+	return aws.AwsRoute53HealthCheckResourceType
+}
+
+func (e *Route53HealthCheckEnumerator) Enumerate() ([]resource.Resource, error) {
+	healthChecks, err := e.repository.ListAllHealthChecks()
+	if err != nil {
+		return nil, remoteerror.NewResourceEnumerationErrorWithType(err, string(e.SupportedType()), aws.AwsRoute53HealthCheckResourceType)
```

Comment:
You should return `remoteerror.NewResourceEnumerationError(err, string(e.SupportedType()))`

Reply:
:+1: